rustutils/Cargo.lock

```diff
@@ -0,0 +1,162 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Is this needed to be checked in

Reply:
@alasram yes, this file is usually checked in for the crates that produce a binary (i.e. an executable, a static or a dynamic library). This way everyone is using exactly the same dependency tree. This file is usually not checked in if its a simple rust library - i.e. something that is used as part of a bigger rust project.